src/helpers/files.ts

```diff
@@ -0,0 +1,15 @@
+import fs from "fs";
```

Comment:
This function is not used anywhere - you can delete this file.

Reply:
done